docs/tutorial/03-model-serving.ipynb

```diff
@@ -17,7 +17,7 @@
     "\n",
     "\n",
     "MLRun serving supports advanced real-time data processing and model serving pipelines.<br>\n",
-    "For more details and examples, see the {ref}`MLRun serving pipelines <serving>` documentation.\n",
+    "For more details and examples, see the {ref}`MLRun serving pipelines <serving-graph>` documentation.\n",
```

Comment:
better to have a full URL in case this NB is downloaded

Reply:
my goal was to fix the warnings, we can replace those in the tutorial notebooks if we think we are not going to break those links

Reply:
the main categories (serving, projects, functions, ..) would always have consistent/stable URL (we should not change the names/paths for those), can add readthedocs redirects for broken links